__tests__/InvalidMutations.ts

```diff
@@ -38,25 +38,20 @@ class R1 extends Reactor {
       [this.in1],
       [this.in1, this.in2, this.out1, this.out2],
       function (this, __in1, __in2, __out1, __out2) {
-        test("expect error on creating creating direct feed through", () => {
-          expect(() => {
-            this.connect(__in2, __out2);
-          }).toThrowError("New connection introduces direct feed through.");
-        });
         test("expect error when creating connection outside container", () => {
           expect(() => {
-            this.connect(__out2, __in2);
+            this.connect(__out2.asConnectable(), __in2.asConnectable());
```

Comment:
If any port can be converted to a connectable one, then what is the purpose? Moreover, the "kind" of port should be specified in the scope of the constructor, when the reaction is added and its arguments are specified...

Reply:
This piece of code is more like a temporary short-hand fix.
The purpose of `connectablePort` is not a safeguard for the users, but rather a convenience for the argument processor. If a normal/`writablePort` is passed in, it will add dependencies against the mutation itself. `connectablePort` wont.
Inside the body, if we were to write typescript, this type of casting is inevitable.